wandb/sdk/internal/internal_api.py

```diff
@@ -408,7 +408,7 @@ def parse_slug(
             assert run, "run must be specified"
         return project, run
 
-    @normalize_exceptions
+    # @normalize_exceptions
     def server_info_introspection(self) -> Tuple[List[str], List[str]]:
```

Comment:
should we remove this completely and just have multiple queries? seems to be safer...